docs/source/guide/kms-example-encrypt-decrypt-file.rst

```diff
@@ -29,9 +29,6 @@ example, with the ``pip`` command.
 
     pip install cryptography
 
-Each section describes a single function from the example's `entire
-source file <https://github.com/awsdocs/aws-doc-sdk-examples/tree/master/python/example_code/kms/encrypt_decrypt_file.py>`_.
-
```

Comment:
We should potentially have @shepazon  or @meyertst-aws do a look over this to see if we need to make other updates. It looks like the samples below are intended to be inline with the linked repo, we've updated the content there but not here.

Reply:
I'll look at this as soon as possible. Hopefully today.

Reply:
I believe this change can be merged and deployed as-is. The existing sample code that's in the guide is still valid. I have filed a ticket that is on my backlog to update the guide content to center on the new examples, however.